internal/services/account_token/testdata/account_token-permissiongroup-order.tf

```diff
@@ -1,22 +1,28 @@
-resource "cloudflare_account_token" "%[1]s" {
```

Comment:
An aside - these `"%[1]s"` resource names are for some reason removed when I save the file in my vscode. Not sure what is causing that but giving it a definite resource name stops that. I see no reason why these can't have definite resource names.